docs/presto-admin-commands.rst

```diff
@@ -131,6 +131,27 @@ Example
     sudo ./presto-admin server status
 
 
+**************
+server upgrade
+**************
+::
+
+    presto-admin server upgrade local_package_path [local_config_dir]
+
+This command upgrades the Presto RPM on all of the nodes in the cluster to the RPM,
```

Comment:
this sentence is confusing.  maybe say "to the rpm specified by local_package_path.